pronto/term.py

```diff
@@ -253,6 +253,13 @@ def objects(self, r: Relationship) -> Iterator["Term"]:
                         yield ont.get_term(other)
             done.add(node)
 
+    def get_uri(self, term, ontology):
+        prefix, local = term.id.split(":")
+        if prefix in ontology.metadata.idspaces:
+            return "{}{}".format(ontology.metadata.idspaces[prefix][0], local)
+        else:
+            return "http://purl.obolibrary.org/obo/{}_{}".format(prefix, local)
```

Comment:
probably not smart to assume OBO URIs for any old ontology. This should raise an exception like a `KeyError` if there's no idspace defined for the prefix.

Reply:
@cthoyt : Not necessarily, because the prefix could also originate from an import. It would